apps/token/src/routes/governance/components/proposals-list-item/proposals-list-item-details.tsx

```diff
@@ -0,0 +1,192 @@
+import classnames from 'classnames';
+import { Link } from 'react-router-dom';
+import { Button, Icon } from '@vegaprotocol/ui-toolkit';
+import { useVoteInformation } from '../../hooks';
+import { useUserVote } from '../vote-details/use-user-vote';
+import {
+  StatusPass,
+  StatusFail,
+} from '../current-proposal-status/current-proposal-status';
+import { format, formatDistanceToNowStrict } from 'date-fns';
+import { useTranslation } from 'react-i18next';
+import { DATE_FORMAT_DETAILED } from '../../../../lib/date-formats';
+import { ProposalState } from '../../../../__generated__/globalTypes';
+import type { ReactNode } from 'react';
+import type { Proposals_proposals } from '../../proposals/__generated__/Proposals';
+
+export const ProposalsListItemDetails = ({
+  proposal,
+}: {
+  proposal: Proposals_proposals;
+}) => {
+  const { state } = proposal;
+  const { willPass, majorityMet, participationMet } = useVoteInformation({
+    proposal,
+  });
+  const { t } = useTranslation();
+  const { voteState } = useUserVote(
+    proposal.id,
+    proposal.votes.yes.votes,
+    proposal.votes.no.votes
+  );
+  const MajorityNotReached = (
```

Comment:
chore (non-blocking): i'd move these two components outside of `ProposalsListItemDetails`, they don't need to be recreated in the scope every time the component rerenders, they can just be defined statically, like:

```
const MajorityNotReached = <>...</>;
const ParticipationNotReached = <>...</>;

const ProposalsListItemDetails () => {...}
```